### DISCUSSION TITLE
Option for ser2net sticks on the network?

### DISCUSSION THREAD

**Discussion by perhusaas** [zwave-js/zwave-js-ui#2558] [Ideas]
I have a common network in several houses (too far apart for z-wave coverage, and i'd like separate z-wave networks anyway).

One way to make contact is ser2net/socat, but I'd imagine this could be better served implemented this package? 

If the option to add a IP/port instead of a /dev/ttyXX address would be awesome!

**Comment by robertsLando** [comment]
> If the option to add a IP/port instead of a /dev/ttyXX address would be awesome!

It's already possible! Just type it in the input, it's not a select but a combobox, it means it allow you to both select from available options or insert a custom one :)

**Comment by perhusaas** [comment]
Perfect! 

Then I guess this topic is about updating the documentation to reflect it? 
Couldn't find any mention of this here? https://zwave-js.github.io/zwavejs2mqtt/#/usage/setup

Thanks to all for creating a great HA product! :)

**Reply by robertsLando** [reply]
> Then I guess this topic is about updating the documentation to reflect it?
Couldn't find any mention of this here? [zwave-js.github.io/zwavejs2mqtt/#/usage/setup](https://zwave-js.github.io/zwavejs2mqtt/#/usage/setup)

Agree, docs are not always up to date, but the hint under the input says: 

`Ex /dev/ttyUSB0. If your port is not listed here just write the port path here`

Will also update docs BTW to make it more clear :)

**Reply by perhusaas** [reply]
Very good. How should it be written? Some need "ws://IP:Port", "socket://" or just "IP:PORT"?

**Reply by robertsLando** [reply]
I think ip:port should be enough, or maybe tcp://, never used it but I now many users used it successfully

**Reply by perhusaas** [reply]
`tcp://<IP>:<PORT>`  works well. Thanks!
